web-console/monitoring/metrics/metrics.md

```diff
@@ -0,0 +1,35 @@
+# Metrics
+
+![Metrics page with an empty query field](img/metrics-empty.png)
```

Comment:
Nits:

- The “5 h” dropdown in the top-left corner, would writing out “hours” be clearer, or does this match something internally?
- What does “Hide chart” do when there isn’t any chart yet? Should it be disabled and/or hidden entirely?

Reply:
The h5 notation is a holdover from prometheus and is present in a few other places. It'll be replaced by the datepicker when that becomes available.

The Hide action would collapse that whole top section to save space when people are only interested in the tables. It'd be useful whether or not a graph was present at the time

Reply:
Should "Hide Graph" / "Show Graph" have icons like the current monitoring UI graphs? (`fa-line-chart` / `fa-compress`)?

Reply:
@kyoto I think that makes sense. I left the icons out in anticipation of switching to pf4, but I think there's a version of a link button with icons now, so we can keep them in the switch.